### DISCUSSION TITLE
Отправить электронное письмо с вложенным файлом.

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#371] [Как это сделать?] [answered]
Есть задача отправить электронное письмо с вложенным файлом ( CSV или другой ). Файл необходимо получить из таблицы TOTUM. Код планирую использовать в КРОНЕ. Помогите с направлением решения.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
```
=: emailSend(to: "test@example.ru"; title: "Отправка письма"; body: "Тестовое письмо"; files: json`[$filerow]`)
    fileRow: rowCreate(field: "name" = "Текст файл.csv"; field: "filestring" = $csv)
        csv: "Здесь генерируем CSV"
```

**Reply by ttm-alexeypolunin** [reply]
https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/298

**Reply by vldekov** [reply]
csv: "Здесь генерируем CSV"
Можно поподробнее это место...

**Reply by ttm-alexeypolunin** [reply]
> #298

вот ссылка на тред

**Reply by ttm-alexeypolunin** [reply]
Если у вас на входе `rowList` то надо по строке склеить и потом по вертикали через `listJoin`.

Но вообще надо добавить функцию, но лениво.
